vector-im/element-android
# [Device Management] Long session names not rendered well (PSG-820)

## Type of change


## Content


Fixing issues in UI when session have long names in the session overview and details screens.

## Motivation and context


Closes #7310 

## Screenshots / GIFs



|Before|After|
|-|-|
|<img src="https://user-images.githubusercontent.com/46314705/194522491-2c3b431d-a21e-42fe-a87d-e198c417661e.png" width=300 />|<img src="https://user-images.githubusercontent.com/46314705/194522659-e2928dc6-9d36-4adc-a81d-fae5c886ff84.png" width=300 />|
|<img src="https://user-images.githubusercontent.com/46314705/194522741-5e2d9a7d-7492-49a0-adc5-f91a0643c3d8.png" width=300 />|<img src="https://user-images.githubusercontent.com/46314705/194522825-be5b22df-946e-4552-9e28-2fd016d4ff90.png" width=300 />|

## Tests



- Enable the new device management feature flag
- Go to Settings -> Security & Privacy -> Show all sessions (WIP, V2)
- Go to session overview screen for a session
- Check the name is well displayed
- Go to session details screen
- Check the display is correct

## Tested devices

- OS version(s): Android 11

## Checklist
